packages/mobile/src/app/App.tsx

```diff
@@ -118,6 +78,29 @@ export class App extends React.Component {
     Linking.removeEventListener('url', this.handleOpenURL)
   }
 
+  getAppStartTimeFromNative(appLoadedTime: number) {
```

Comment:
nit: Can we rename to `logAppLoadDuration`? I think `get` implies that we expect the function to return something whereas `log` makes it clearer that we are only logging the results